pkg/provider/aws/provider.go

```diff
@@ -49,3 +51,56 @@ func (a *AWS) GetResourcesDirectlyTiedToConstruct(construct core.BaseConstruct)
 	resources, found := a.constructIdToResources[construct.Id()]
 	return resources, found
 }
+
+func (a *AWS) LoadGraph(graph core.OutputGraph, dag *core.ConstructGraph) error {
```

Comment:
It'd be nice to have a unit test for this, if only to see what the format is and do a quick smoke test.

Reply:
yeah i can add one for it